adapters/rubicon/rubicon.go

```diff
@@ -946,9 +946,7 @@ func extractUserBuyerUID(eids []openrtb2.EID) string {
 				continue
 			}
 
-			if uidExt.Stype == "ppuid" || uidExt.Stype == "other" {
```

Comment:
Can we remove `rubiconUserExtEidUidExt` entirely if it's not used anymore in the adapter code?